packages/jmix-react-antd/src/ui/form/controls/RichTextField.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import { Editor } from "react-draft-wysiwyg";
+import styles from "./RichTextArea.module.less";
+import classNames from "classnames";
+import { getMainStore } from "@haulmont/jmix-react-core";
+import { observer } from "mobx-react";
+import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
+import draftToHtml from "draftjs-to-html";
+import htmlToDraft from "html-to-draftjs";
+import { RawDraftContentState, EditorState, ContentState } from "draft-js";
+
+export interface RichTextFieldProps {
+  isReadOnly: boolean;
+  onChange?: Function;
+  value?: string;
+}
+
+export const RichTextField = observer(
+  ({ isReadOnly, onChange, value, ...rest }: RichTextFieldProps) => {
+    const mainStore = getMainStore();
+
+    const handleChange = React.useCallback(
+      (state: RawDraftContentState) => {
+        if (onChange) onChange(rawStateToString(state));
+      },
+      [onChange]
+    );
+
+    function stringToEditorState(input: string): EditorState {
+      const { contentBlocks, entityMap } = htmlToDraft(input);
+      return EditorState.createWithContent(
+        ContentState.createFromBlockArray(contentBlocks, entityMap)
+      );
+    }
+
+    function rawStateToString(state: RawDraftContentState): string {
+      return draftToHtml(state);
+    }
```

Comment:
Please move the helper functions outside of the component definition, otherwise they will be recreated on each render.